.github/workflows/ci.yml

```diff
@@ -117,8 +117,11 @@ jobs:
       - name: Setup the test MLWH and Events databases
         run: |
           python setup_test_db.py
+      - name: Start a testing SFTP server
+        run: |
+          docker run -p "3010:22" -v ${PWD}/test:/home/foo -d atmoz/sftp foo:pass:1001
```

Comment:
Just to clarify, the reason why we start the SFTP service here instead of in the services section of the github actions file, is because we don't have the folder we want to setup in the SFTP until we have downloaded the code from git, and that only happens after all services have started.